packages/editor-ui/src/stores/__tests__/assistant.store.test.ts

```diff
@@ -428,8 +429,7 @@ describe('AI Assistant store', () => {
 		const message = assistantStore.chatMessages[0];
 		expect(message.type).toBe('error');
 
-		// @REVIEW How to best get the type from design-system in here?
-		const errorMessage = message as { retry?: () => Promise<void> };
+		const errorMessage = message as ChatUI.ErrorMessage;
```

Comment:
try to avoid type coercions (X as Y for example) but since it's in tests, it's fine

Reply:
Yes, ideally the testing framework would apply this coercion from the expect above, but the call pattern is a bit too awkward. I could alternatively throw in an explicit assert.